bom-internal/pom.xml

```diff
@@ -484,6 +472,18 @@
                 <artifactId>plexus-utils</artifactId>
                 <version>3.0.24</version>
             </dependency>
+            <dependency>
+                <groupId>net.bytebuddy</groupId>
+                <artifactId>byte-buddy</artifactId>
+                <version>${bytebuddy.version}</version>
```

Comment:
Do we need to add test scope?

Reply:
Technically shouldn't matter since this is in the BOM but I'll add it!